libs/e2e/src/lib/describer/modelAssessment/modelOverview/ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent.ts

```diff
@@ -23,7 +24,10 @@ export function ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent(
     "not.exist"
   );
   if (isNotebookTest) {
-    if (getNumberOfCohorts(datasetShape, includeNewCohort) <= 1) {
+    if (
+      getNumberOfCohorts(datasetShape, includeNewCohort) <= 1 ||
+      datasetShape.isObjectDetection === true
```

Comment:
nit: remove === true on boolean, so just:
```
|| datasetShape.isObjectDetection
```

Reply:
you can try !!datasetShape.isObjectDetection, this converts string/undefined to boolean. Direct use of datasetShape.isObjectDetection works too.